### PR TITLE
CSV loader test modification 

### PR DESCRIPTION
**Pull request by qlinsey**
1. removed 2 test cases from CoolModelTest 
2. added separate test cases for CSV loader 
3. Please give feedback and will update accordingly.

### CODE REVIEW

File: cool-core/src/test/java/com/nus/cool/functionality/CsvProperties.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+        // get the property value and print it out
+        System.out.println(prop.getProperty("cube"));
+        System.out.println(prop.getProperty("schemaFileName"));
+        System.out.println(prop.getProperty("dimFileName"));
+        System.out.println(prop.getProperty("dataFileName"));
+        System.out.println(prop.getProperty("cubeRepo"));
+        return prop;
+    }
+    public static void main(String[] args) throws IOException {
```
Comment (Zrealshadow) [inline cool-core/src/test/java/com/nus/cool/functionality/CsvProperties.java:47]: Do not add executable program in test directory.

File: cool-core/src/test/java/com/nus/cool/functionality/csv.properties [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#//Nomral case
+cube_0 = health
```
Comment (Zrealshadow) [inline cool-core/src/test/java/com/nus/cool/functionality/csv.properties:2]: We put static data under `resources` directory

File: cool-core/src/test/java/com/nus/cool/functionality/CsvProperties.java (hunk 1 above)
Comment (qlinsey) [reply cool-core/src/test/java/com/nus/cool/functionality/CsvProperties.java:47]: yes, it is for my side testing. I will remove later.

File: cool-core/src/test/java/com/nus/cool/functionality/csv.properties (hunk 2 above)
Comment (qlinsey) [reply cool-core/src/test/java/com/nus/cool/functionality/csv.properties:2]: i can move under resources

**Pull request closed without merging**